Marlin/src/HAL/NATIVE_SIM/u8g/u8g_com_st7920_sw_spi.cpp

```diff
@@ -71,33 +71,33 @@ static uint8_t SPI_speed = 0;
 
 static uint8_t swSpiTransfer(uint8_t b, const uint8_t spi_speed, const pin_t sck_pin, const pin_t miso_pin, const pin_t mosi_pin) {
   for (uint8_t i = 0; i < 8; i++) {
-    WRITE_PIN(mosi_pin, !!(b & 0x80));
+    WRITE_PIN(sck_pin, TERN(U8G_SPI_USE_MODE_3, LOW, HIGH));
     DELAY_CYCLES(SPI_SPEED);
-    WRITE_PIN(sck_pin, HIGH);
+    WRITE_PIN(mosi_pin, !!(b & 0x80));
     DELAY_CYCLES(SPI_SPEED);
     b <<= 1;
     if (miso_pin >= 0 && READ_PIN(miso_pin)) b |= 1;
-    WRITE_PIN(sck_pin, LOW);
+    WRITE_PIN(sck_pin, TERN(U8G_SPI_USE_MODE_3, HIGH, LOW));
     DELAY_CYCLES(SPI_SPEED);
   }
   return b;
 }
 
 static uint8_t swSpiInit(const uint8_t spiRate, const pin_t sck_pin, const pin_t mosi_pin) {
   WRITE_PIN(mosi_pin, HIGH);
-  WRITE_PIN(sck_pin, LOW);
+  WRITE_PIN(sck_pin, TERN(U8G_SPI_USE_MODE_3, HIGH, LOW));
```

Comment:
Seems like SCK should be init'ed to the opposite state from Mode 0 too, otherwise one clock tick, and thus one bit, may be lost.